docs/configure/sidebar-and-urls.md

```diff
@@ -137,52 +137,55 @@ When Storybook generates the titles for all matching stories, they'll retain the
 
 ### Story Indexers
 
-Story Indexers are usually responsible of crawling through your filesystem on your given glob path, and retrieve the stories that match that glob, afterwards Storybook analyzes these stories and create entries for these stories in the `index.json` (formerly `stories.json`). This `index.json` is used to populate the sidebar links based on the `title` retrieved for each story from the story file.
-
-For CSF, it is either auto generated or retrieved from the meta configuration.
+Story Indexers are a set of heuristics used by Storybook to crawl your filesystem based on a given glob pattern searching for matching stories, which is then used to generate an index.json (formerly stories.json) file responsible for populating the sidebar with the necessary information. By default, this heuristic will look for files that contain the following scheme \*.stories.@(js|jsx|ts|tsx). However, if you need, you can create your custom story indexer that you can use to include stories that have a different naming convention. For example:
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'common/csf-3-example-title.ts.mdx',
+    'common/storybook-main-csf-indexer.js.mdx',
   ]}
```

Comment:
@sheriffMoose , one item here that requires your attention, if you're ok with it, here you are referencing the file "storybook-main-csf-indexer.js.mdx", but looking at the snippet files, that particular one was renamed to "storybook-main-csf-indexer.ts.mdx", which would lead to it not being loaded at all as the snippet doesn't exist. We have two options:
 - Add the JS version if it works for this scenario and reference both, turning it into:
```js
<!-- prettier-ignore-start -->

<CodeSnippets
  paths={[
    'common/storybook-main-csf-indexer.js.mdx',
    'common/storybook-main-csf-indexer.ts.mdx',
  ]}
/>

<!-- prettier-ignore-end -->
```
- Update the snippet reference in the documentation to only point at the TypeScript version, turning into:
```js
<!-- prettier-ignore-start -->

<CodeSnippets
  paths={[
    'common/storybook-main-csf-indexer.ts.mdx',
  ]}
/>

<!-- prettier-ignore-end -->
```

Reply:
done, thanks